libsolidity/interface/CompilerStack.cpp

```diff
@@ -841,3 +842,81 @@ string CompilerStack::computeSourceMapping(eth::AssemblyItems const& _items) con
 	}
 	return ret;
 }
+
+namespace {
+
+Json::Value gasToJson(GasEstimator::GasConsumption const& _gas)
+{
+    if (_gas.isInfinite || _gas.value > std::numeric_limits<Json::LargestUInt>::max())
+	return Json::Value(Json::nullValue);
```

Comment:
Hm, I actually think `null` fits better here - we can still change the documentation, can't we?